rajawali/src/main/java/org/rajawali3d/loader/LoaderOBJ.java

```diff
@@ -416,7 +416,12 @@ private void mergeGroupsAsObjects(Object3D object) {
 	}
 
 	private static String generateObjectName() {
-		return "Object" + (int) (Math.random() * 10000);
+		return generateObjectName(null);
+	}
+
+	private static String generateObjectName(String prefix) {
+		if(prefix==null) prefix = "Object";
+		return prefix + (int) (Math.random() * 9e3 + 1e3);
```

Comment:
Why not using a timestamp (or UUID) ?

Reply:
Excellent idea, I went with a timestamp as the ramifications of that system call is better understood.

Reply:
Seeing this now after my other question. I really think uuid implies the intent vs a random number or nanos usage.